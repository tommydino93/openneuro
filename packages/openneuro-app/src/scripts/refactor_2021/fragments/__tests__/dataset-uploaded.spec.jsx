import React from 'react'
import { shallow } from 'enzyme'
import DatasetUploaded from '../dataset-uploaded.jsx'

describe('DatasetUploaded component', () => {
  it('renders with common props', () => {
    const wrapper = shallow(
      <DatasetUploaded
        uploader={{ name: 'Tester' }}
        created="2016-11-06"
        testDifference="awhile ago"
      />,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
