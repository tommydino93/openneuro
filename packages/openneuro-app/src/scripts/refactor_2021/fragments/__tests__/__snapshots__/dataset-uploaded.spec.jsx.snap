// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatasetUploaded component renders with common props 1`] = `
<h6>
  uploaded by Tester on 2016-11-06 - awhile ago ago
</h6>
`;
